src/main/java/gregtech/common/metatileentities/steam/SteamMiner.java

```diff
@@ -135,22 +135,23 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
     }
 
     void addDisplayText(List<ITextComponent> textList) {
+        int workingArea = getWorkingArea(minerLogic.getCurrentRadius());
         textList.add(new TextComponentTranslation("gregtech.machine.miner.startx", this.minerLogic.getX().get()));
         textList.add(new TextComponentTranslation("gregtech.machine.miner.starty", this.minerLogic.getY().get()));
         textList.add(new TextComponentTranslation("gregtech.machine.miner.startz", this.minerLogic.getZ().get()));
-        textList.add(new TextComponentTranslation("gregtech.machine.miner.radius", this.minerLogic.getCurrentRadius()));
+        textList.add(new TextComponentTranslation(I18n.format("gregtech.universal.tooltip.working_area", workingArea, workingArea)));
```

Comment:
This call to I18n.format() is unneeded, and will likely crash on a dedicated server.